content/en/about/security-model/index.md

```diff
@@ -47,9 +47,9 @@ HUGO_SECURITY_HTTP_URLS=none hugo
 
 ## Dependency Security
 
-Hugo builds as a static binary using [Go Modules](https://github.com/golang/go/wiki/Modules) to manage its dependencies. Go Modules have several safeguards, one of them being the `go.sum` file. This is a database of the expected cryptographic checksums of all of your dependencies, including any transitive.
+Hugo is built as a static binary using [Go Modules](https://github.com/golang/go/wiki/Modules) to manage its dependencies. Go Modules have several safeguards, one of them being the `go.sum` file. This is a database of the expected cryptographic checksums of all of your dependencies, including transitive dependencies.
 
-[Hugo Modules](/hugo-modules/) is built on top of Go Modules functionality, and a Hugo project using Hugo Modules will have a `go.sum` file. We recommend that you commit this file to your version control system. The Hugo build will fail if there is a checksum mismatch, which would be an indication of [dependency tampering](https://julienrenaux.fr/2019/12/20/github-actions-security-risk/).
+[Hugo Modules](/hugo-modules/) are built on top of the functionality of Go Modules, and a Hugo project using Hugo Modules will have a `go.sum` file. We recommend that you commit this file to your version control system. The Hugo build will fail if there is a checksum mismatch, which would be an indication of [dependency tampering](https://julienrenaux.fr/2019/12/20/github-actions-security-risk/).
```

Comment:
```suggestion
[Hugo Modules](/hugo-modules/) is built on top of the functionality of Go Modules, and a Hugo project using Hugo Modules will have a `go.sum` file. We recommend that you commit this file to your version control system. The Hugo build will fail if there is a checksum mismatch, which would be an indication of [dependency tampering](https://julienrenaux.fr/2019/12/20/github-actions-security-risk/).
```

Reply:
"Hugo Modules" is a feature (one thing, singular).

Reply:
A feature can be consisting of multiple things. Maybe we should try to have documentation that is understood by not only English Majors but also normal people. Changing this specific wording from are to is will irritate readers. On the other side, if this is a forced concoction it should be phrased as "Hugo Modules is a feature that is ..." instead of "Hugo Modules is ..."

To compare: The "Go Modules" documentation has no issue to talk about a singular module: https://go.dev/blog/using-go-modules

Reply:
The "is" read a bit weird to me at first. Then I thought the same thing that was mentioned above, that it was the name of a feature. However, the actual [Hugo Modules](https://gohugo.io/hugo-modules/) page it links to uses "Hugo Modules are" in multiple places. That prompted me to make the change.

Reply:
I thought we had consensus on this:
<https://github.com/gohugoio/hugoDocs/pull/1266#issuecomment-713394631>

Reply:
In that case, I like what @davidsneighbour proposed with the "Hugo Modules is a feature that is ..." wording. Especially since in the context of this specific page this is the first time the Hugo Modules feature is being introduced to the reader.

Reply:
@sudomateo Works for me. We have a similar feature/name issue with Hugo Pipes...

Reply:
Updated!